### DISCUSSION TITLE
Backport fix to remove startup...  clean builds fail because startup crate is missing

### DISCUSSION THREAD

**Discussion by golfingal72** [apollographql/router#1710] [Q&A] [answered]
Is there any way to backport the fixes for the crate.io removing 'startup' and replacing with 'ctor'?  

The 1.0.x versions are labeled alpha and I want to be able to use the 0.15.x preferrably, but will use the 0.16.0 if i have to

**Comment by abernix** [comment, marked as answer]
Good question!  Thanks for bringing this up.

Our project uses [semantic versioning](https://semver.org/) as the guiding principle for our releases and the meaning of each release number.  Per that specification, everything prior to 1.0.0 is experimental _at best_.  This means that the 1.0.0-alpha releases are **substantially more stable** than 0.16.x, 0.15.x or any lower pre-0.x version.

Put another way, we're not going to back-port the fixes to versions that were experimental.  We're no longer providing updates for them and the guidance is to update to a 1.0.0 version as soon as possible.   Our intention is to graduate the `1.0.0-alpha` versions into a "release candidate" this week.  The "release candidate" will receive any necessary bug fixes prior to us shipping it as an 1.0.0 (without any designator) in the coming weeks.

**Reply by abernix** [reply]
Aside from perceived stability, I'm curious if there is anything else preventing you from upgrading to Router 1.0.0-alpha?

**Reply by golfingal72** [reply]
I ended up going up through the chain to upgrade to the alpha.2,   I'll be on the lookout for the "official" 1.0.0 release...  so I marked as answered...
